isBrowser is false as of webpack@4.16.2

for example on the recent version of webpack we were getting:

error like this:
```
WARNING in ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js

Module not found: Error: Can't resolve 'gl' in '/home/jenkins/workspace/.../node_modules/luma.gl/dist/esm/webgl-context'

 @ ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js

 @ ./node_modules/luma.gl/dist/esm/webgl-context/context.js

 @ ./node_modules/luma.gl/dist/esm/index.js

 @ ./node_modules/@deck.gl/core/dist/esm/lib/layer-manager.js

 @ ./node_modules/@deck.gl/core/dist/esm/index.js

 @ ./node_modules/deck.gl/dist/esm/index.js

```
and it looks like isBrowser from https://github.com/uber/luma.gl/blob/v6.0.0/src/utils/is-browser.js#L12 is false 

related https://github.com/webpack/webpack/issues/7777

still seeing this with `luma.gl@6.0.0`

```
WARNING in ./node_modules/probe.gl/dist/lib/log.js
Module not found: Error: Can't resolve 'asciify-image' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/probe.gl/dist/lib'
 @ ./node_modules/probe.gl/dist/lib/log.js
 @ ./node_modules/probe.gl/dist/index.js
 @ ./node_modules/@deck.gl/core/dist/esm/utils/log.js
 @ ./node_modules/@deck.gl/core/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...

WARNING in ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js
Module not found: Error: Can't resolve 'gl' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/luma.gl/dist/esm/webgl-context'
 @ ./node_modules/luma.gl/dist/esm/webgl-context/create-headless-context.js
 @ ./node_modules/luma.gl/dist/esm/webgl-context/index.js
 @ ./node_modules/luma.gl/dist/esm/core/animation-loop-proxy.js
 @ ./node_modules/luma.gl/dist/esm/index.js
 @ ./node_modules/@deck.gl/layers/dist/esm/hexagon-cell-layer/hexagon-cell-layer.js
 @ ./node_modules/@deck.gl/layers/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...

WARNING in ./node_modules/luma.gl/dist/es5/webgl1/webgl-rendering-context.js
Module not found: Error: Can't resolve 'gl/wrap' in '/Users/craigbeck/dev/premise-frontend-v3/node_modules/luma.gl/dist/es5/webgl1'
 @ ./node_modules/luma.gl/dist/es5/webgl1/webgl-rendering-context.js
 @ ./node_modules/luma.gl/dist/es5/webgl1/polyfill-context.js
 @ ./node_modules/luma.gl/dist/es5/webgl1/index.js
 @ ./node_modules/luma.gl/webgl1.js
 @ ./node_modules/@deck.gl/core/dist/esm/lib/init.js
 @ ./node_modules/@deck.gl/core/dist/esm/index.js
 @ ./node_modules/deck.gl/dist/esm/index.js
...
```

--- a/package.json
+++ b/package.json
@@ -37,6 +37,10 @@
     "headless.js",
     "webgl1.js"
   ],
+  "browser": {
+    "gl": false,
+    "gl/wrap": false
+  },
   "scripts": {
     "start": "echo 'Please see luma.gl website for how to run examples' && open http://uber.github.io/luma.gl/#/documentation/getting-started/examples",
     "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5/packages dist/esm/packages dist/es6/packages",

--- a/package.json
+++ b/package.json
@@ -73,7 +73,7 @@
   },
   "dependencies": {
     "math.gl": "^2.0.0",
-    "probe.gl": "^1.0.0",
+    "probe.gl": "^1.0.4",
     "seer": "^0.2.4",
     "webgl-debug": "^2.0.0"
   },

--- a/src/webgl-context/create-headless-context.js
+++ b/src/webgl-context/create-headless-context.js
@@ -1,7 +1,4 @@
-/* eslint-disable quotes, no-console */
-import isBrowser from '../utils/is-browser';
-
-import {ERR_HEADLESSGL_LOAD} from '../webgl-utils/webgl-types';
+import {headlessGL} from '../webgl-utils/webgl-types';
 
 const ERR_HEADLESSGL_NOT_AVAILABLE =
 'Failed to create WebGL context in Node.js, headless gl not available';
@@ -11,15 +8,6 @@ const ERR_HEADLESSGL_FAILED =
 
 // Create headless gl context (for running under Node.js)
 export function createHeadlessContext({width, height, opts, onError}) {
-  let headlessGL;
-  if (!isBrowser) {
-    try {
-      headlessGL = module.require('gl');
-    } catch (error) {
-      return onError(`${ERR_HEADLESSGL_LOAD}\n${error}`);
-    }
-  }
-
   const {webgl1, webgl2} = opts;
   if (webgl2 && !webgl1) {
     return onError('headless-gl does not support WebGL2');

--- a/src/webgl-utils/webgl-types.js
+++ b/src/webgl-utils/webgl-types.js
@@ -17,9 +17,13 @@ If this is desired, install headless gl using "npm install gl --save-dev" or "ya
 
 // Load headless gl dynamically, if available
 export let headlessTypes = null;
+export let headlessGL = () => {
+  throw new Error(ERR_HEADLESSGL_LOAD);
+};
 
-if (!isBrowser) {
+if (!isBrowser && module.require) {
   try {
+    headlessGL = module.require('gl');
     headlessTypes = module.require('gl/wrap');
   } catch (error) {
     console.info(ERR_HEADLESSGL_LOAD);

